router/router.py

```
@@ -0,0 +1,262 @@
+        message: Message dictionary with optional TTL
+        
+    Returns:
+        bool: True if message has expired
+    """
+    ttl = message.get('metadata', {}).get('ttl')
+    if not ttl:
+        return False
+        
+    try:
+        ttl_dt = datetime.fromisoformat(ttl.replace('Z', '+00:00'))
+        return datetime.now(tll_dt.tzinfo) > ttl_dt
```

Ruff (0.11.9)

Undefined name `tll_dt`

(F821)